hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMHAConfiguration.java

```diff
@@ -114,6 +114,14 @@
   )
   private int raftLogPurgeGap = 1000000;
 
+  @Config(key = "ratis.snapshot.threshold",
+      type = ConfigType.LONG,
+      defaultValue = "100000L",
+      tags = {SCM, OZONE, HA, RATIS},
+      description = "The threshold to trigger a Ratis taking snapshot " +
+          "operation")
+  private long ratisSnapshotThreshold = 100000L;
```

Comment:
100000L might be too large for SCM, since the WriteBatch needs hold them in memory and the flush will block the apply of transactions. How about 500 or 1000 ?

Reply:
make senses. Set as 1000 for now. This is a configurable value anyway.